packages/vscode-extension/src/test/suite/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as path from 'path';
+
+import * as Mocha from 'mocha';
+import * as glob from 'glob';
+
+export async function run(): Promise<void> {
+  // Create the mocha test
+  const mocha = new Mocha({
```

Comment:
We need to use Mocha instead of Jest for the plugins?

Reply:
I'll change to Jest, but looks like mocha is the "recommended" test tool from VSCode team.